### PR TITLE
Upgrade to lsstts/develop-env:c0018.000

### PR DESCRIPTION
**Pull request by sebastian-aranda**
A new development environment was released, so this PR updates the image and make some adjustments to comply with the new version.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ pipeline {
     registryCredential = "dockerhub-inriachile"
     dockerImageName = "lsstts/love-commander:"
     dockerImage = ""
-    LSSTTS_DEV_VERSION = "c0016.001"
+    LSSTTS_DEV_VERSION = "c0018.000"
```
Comment (areyesd14) [inline Jenkinsfile:7]: In this line, It should be `dev-cycle` variable?

Comment (sebastian-aranda) [reply Jenkinsfile:7]: Thanks for the review @areyesd14 ! Well, this is part of the things that we haven't extended yet. The Jenkinsfile can't read the `.env` file with the respective environment variables, opposed to the docker-compose that implicit loads the environment variables of the `.env` file. So **no**, it has to be hardcoded until we find a method to extend this.

Comment (areyesd14) [reply Jenkinsfile:7]: Ahh, that makes sense.

**Review by areyesd14 — APPROVED** [review]
Great job! Thanks!

**Pull request merged**